utils/secrets.go

```diff
@@ -94,6 +95,17 @@ func (s V2) GetVersion() string {
 	return "2"
 }
 
+// Find searches the envelope for a named string. Returns the index of the element found
+// in the envelope or -1 if not found.
+func (s *Secrets) Find(secretName string) (int, error) {
+	for i, n := range *s {
+		if secretName == n.Name {
+			return i, nil
+		}
+	}
+	return 0, errors.New("Secret not found")
+}
```

Comment:
based on the usage of this, is there a reason for having this return the index as opposed to the secret itself? I feel like if you are given the index, you will always be accessing the secret with that index afterwards, so why not just have that be included in this Find function

Reply:
Good points. :)  :bowing_man:

Reply:
Yeah, much easier to just work with the returned object than having the extra step of assignment post find.